Cannot build ./src/routes/_utils/blurhash.js

During the ```yarn build``` step I'm receiving the following error on an arm64 Docker environment. Is this a known issue or there a work around?

```

ERROR in ./src/routes/_utils/blurhash.js
Module not found: Error: Can't resolve 'worker-loader' in '/app'
 @ ./src/routes/_utils/blurhash.js 1:0-49 21:43-57
 @ ./src/routes/_actions/createMakeProps.js
 @ ./src/routes/_components/timeline/Timeline.html
 @ ./src/routes/_utils/asyncModules.js
 @ ./src/routes/_components/snackbar/snackbar.js
 @ ./src/routes/_utils/serviceWorkerClient.js
 @ ./src/client.js

ERROR in chunk Timeline
52d4d50e86f083bf96db/39.js
/app/node_modules/svelte-loader/index.js??ref--4!/app/src/routes/_components/timeline/Timeline.html c208b4872bd08b052f8cf49e1bd8123e
Unexpected token (22:77)
|
| function init () {
|   worker = worker || new __WEBPACK_MODULE_REFERENCE__29_64656661756c74__(new !(function webpackMissingModule() { var e = new Error("Cannot find module '../_workers/blurhash'"); e.code = 'MODULE_NOT_FOUND'; throw e; }())())
| }
|
> Encountered errors while building app

```

I tried to build a new docker image and it fails  because of blurhash too.
```
ERROR in ./src/routes/_utils/blurhash.js
Module not found: Error: Can't resolve 'worker-loader' in '/opt/pinafore'
 @ ./src/routes/_utils/blurhash.js 1:0-49 21:43-57
 @ ./src/routes/_actions/createMakeProps.js
 @ ./src/routes/_components/timeline/Timeline.html
 @ ./src/routes/_utils/asyncModules.js
 @ ./src/routes/_components/snackbar/snackbar.js
 @ ./src/routes/_utils/serviceWorkerClient.js
 @ ./src/client.js

ERROR in chunk Timeline
ffcdba4492ca40ca1f36/39.js
/opt/pinafore/node_modules/svelte-loader/index.js??ref--4!/opt/pinafore/src/routes/_components/timeline/Timeline.html ea69c9128fd76f18b9643e38dc854a53
Unexpected token (22:77)
| 
| function init () {
|   worker = worker || new __WEBPACK_MODULE_REFERENCE__29_64656661756c74__(new !(function webpackMissingModule() { var e = new Error("Cannot find module '../_workers/blurhash'"); e.code = 'MODULE_NOT_FOUND'; throw e; }())())
| }
| 
> Encountered errors while building app
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "sapper-export" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-steps" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c apk add --update --no-cache --virtual build-dependencies git python build-base clang    && apk add --update --no-cache nodejs npm       && npm i yarn -g        && yarn --production --pure-lockfile    && yarn build    && yarn cache clean     && rm -rf ./src         && apk del build-dependencies' returned a non-zero code: 1
```

--- a/package.json
+++ b/package.json
@@ -103,7 +103,8 @@
     "text-encoding": "^0.7.0",
     "tiny-queue": "^0.2.1",
     "webpack": "^4.39.2",
-    "webpack-bundle-analyzer": "^3.4.1"
+    "webpack-bundle-analyzer": "^3.4.1",
+    "worker-loader": "^2.0.0"
   },
   "devDependencies": {
     "assert": "^2.0.0",
@@ -112,8 +113,7 @@
     "mocha": "^6.2.0",
     "now": "^16.1.1",
     "standard": "^14.0.2",
-    "testcafe": "^1.4.1",
-    "worker-loader": "^2.0.0"
+    "testcafe": "^1.4.1"
   },
   "engines": {
     "node": ">= 8"

